src/main/java/de/zalando/aruha/nakadi/repository/kafka/KafkaTopicRepository.java

```diff
@@ -210,7 +225,8 @@ public TopicPartition getPartition(final String topicId, final String partition)
             topicPartition.setOldestAvailableOffset(toNakadiOffset(consumer.position(tp)));
```

Comment:
As I remember the team agreed that when partition is empty we expose offsets as (0, BEGIN)